docs/kiali/03-01-kiali.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Access Kiali
+type: Details
+---
+
+Kiali can be easily accessed from the left menu on the [Kyma Console](/components/console/#overview-overview) by clicking on the `Service Mesh` item:
```

Comment:
I'm not sure if it is really relevant to have screenshots how to find the kiali link and how the entry page looks like. Maybe @bszwarc can advice better her. However, interesting might be the entry URL. That it starts with kiali.xxx always.

Reply:
I removed the console screenshot and kept the kiali default dashboard and graph as examples of how it looks like.